src/index.js

```diff
@@ -229,6 +241,21 @@ TabGroups.prototype = {
   },
 
   onGroupClose: function(event) {
+    let groupTabCount = this._tabs.getGroupTabCount(
+      this._getTabBrowser(),
+      event.groupID
+    );
+
+    if (groupTabCount > 0) {
```

Comment:
I guess we should remove the annoying alert box since we now have that customizable timeout.